rwengine/src/script/modules/GTA3Module.cpp

```diff
@@ -909,6 +909,9 @@ GTA3Module::GTA3Module() : ScriptModule("GTA3") {
     bind(0x0453, 4, opcode_0453);
     bind(0x0454, 3, opcode_0454);
     bind(0x0455, 3, opcode_0455);
+    bind(0x0456, 1, opcode_0456);
```

Comment:
0456-0458 do exist in GTA III and probably working, but unused in main.scm

Reply:
These seem to be a part of the "gameplay" project, even.